src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-common.sch

```diff
@@ -211,22 +211,17 @@
 
 
     <!-- Creation/revision dates -->
-    <sch:rule context="//gmd:identificationInfo/*/gmd:citation/gmd:CI_Citation
-            |//*[@gco:isoType='gmd:MD_DataIdentification']/gmd:citation/gmd:CI_Citation
-            |//*[@gco:isoType='srv:SV_ServiceIdentification']/gmd:citation/gmd:CI_Citation">
+    <sch:rule context="//gmd:identificationInfo/*/gmd:citation/gmd:CI_Citation/gmd:date/gmd:CI_Date[gmd:dateType/gmd:CI_DateTypeCode/@codeListValue = 'RI_367']/gmd:date
+    |//*[@gco:isoType='gmd:MD_DataIdentification']/gmd:citation/gmd:CI_Citation/gmd:date/gmd:CI_Date[gmd:dateType/gmd:CI_DateTypeCode/@codeListValue = 'RI_367']/gmd:date
+    |//*[@gco:isoType='srv:SV_ServiceIdentification']/gmd:citation/gmd:CI_Citation/gmd:date/gmd:CI_Date[gmd:dateType/gmd:CI_DateTypeCode/@codeListValue = 'RI_367']/gmd:date">
 
-      <sch:let name="missingPublication" value="count(gmd:date[gmd:CI_Date/gmd:dateType/gmd:CI_DateTypeCode/@codeListValue = 'RI_367']) = 0" />
-
-      <sch:assert
-        test="not($missingPublication)"
-      >$loc/strings/PublicationDate</sch:assert>
-
-      <sch:let name="missingCreation" value="count(gmd:date[gmd:CI_Date/gmd:dateType/gmd:CI_DateTypeCode/@codeListValue = 'RI_366']) = 0" />
-
-      <sch:assert
-        test="not($missingCreation)"
-      >$loc/strings/CreationDate</sch:assert>
+      <sch:let name="creationDate" value="../../../gmd:date/gmd:CI_Date[gmd:dateType/gmd:CI_DateTypeCode/@codeListValue = 'RI_366']/gmd:date[1]/gco:Date[1]
```

Comment:
because I am sure the specs which first index to follow. @josegar74 would you confirm about it if it's the gmddate[1] or gco:date[1]. 

But doing gmd:date[1]/gco:Date[1] causes no harm.